scilpy/tractanalysis/fibertube_scoring.py

```diff
@@ -91,27 +155,43 @@ def true_max_voxel(diagonal):
 
 
 @njit
-def get_external_distance_vec(vector, rp, rq):
-    # Given a distance vector between two fiber centroids, find their
-    # external distance
+def get_external_distance_vec(vector, r1, r2):
```

Comment:
`get_external_distance_from_vec` would be clearer

Reply:
I think it's much clearer now :)